ci/plan.yml

```diff
@@ -148,168 +146,7 @@ SUITES:
-          - TestBackendOps.testConv3dTranspose
-          - TestBackendOps.testConvParameterRankExceptions
-          - TestBackendOps.testCastToInt
-          - TestBackendOps.testCastToFloat
-          - TestBackendOps.testCastToUInt
-          - TestBackendOps.testCeil
-          - TestBackendOps.testCumProd
-          - TestBackendOps.resnetLayer1
-          - TestBackendOps.resnetLayer2
-          - TestBackendOps.res2a_branch2a
-          - TestBackendOps.res2a_branch2b
-          - TestBackendOps.res3a_branch2a
-          - TestBackendOps.fc_1000
-          - TestBackendOps.bigMatMul
-          - TestBackendOps.testDupOutputs
-          - TestBackendOps.testBatchDot
-          - TestBackendOps.testBatchDot3
-          - TestBackendOps.testCategoricalCrossentropyLogits
-          - TestBackendOps.testClearSession
-          - TestBackendOps.testConcatenate
-          - TestBackendOps.testConcatenateSymbolic
-          - TestBackendOps.testCumSum
-          - TestBackendOps.testExpandDims
-          - TestBackendOps.testEqual
-          - TestBackendOps.testEqualVersusNumeric
-          - TestBackendOps.testGreater
-          - TestBackendOps.testGreaterEqual
-          - TestBackendOps.testGreaterEqualVersusNumeric
-          - TestBackendOps.testGreaterVersusNumeric
-          - TestBackendOps.testLess
-          - TestBackendOps.testLessEqual
-          - TestBackendOps.testLessEqualVersusNumeric
-          - TestBackendOps.testLessVersusNumeric
-          - TestBackendOps.testNotEqual
-          - TestBackendOps.testNotEqualVersusNumeric
-          - TestBackendOps.testOneHot
-          - TestBackendOps.testOnesLike
-          - TestBackendOps.testPermuteDimensions
-          - TestBackendOps.testRepeat
-          - TestBackendOps.testRepeatElements
-          - TestBackendOps.testResizeImages
-          - TestBackendOps.testResizeImagesBilinear
-          - TestBackendOps.testResizeVolumes
-          - TestBackendOps.testReverse
-          - TestBackendOps.testSeparableConv2d
-          - TestBackendOps.testSliceEllipsis
-          - TestBackendOps.testSliceMessy
-          - TestBackendOps.testSliceShort
-          - TestBackendOps.testSpatial2DPadding
-          - TestBackendOps.testSpatial3DPadding
-          - TestBackendOps.testStack
-          - TestBackendOps.testStd
-          - TestBackendOps.testTemporalPadding
-          - TestBackendOps.testTile
-          - TestBackendOps.testTranspose
-          - TestBackendOps.testTruncatedNormalMean
-          - TestBackendOps.testTruncatedNormalDev
-          - TestBackendOps.testRandomNormalDev
-          - TestBackendOps.testRandomUniformMean
-          - TestBackendOps.testRandomUniformDev
-          - TestBackendOps.testRandomUniformVariableMean
-          - TestBackendOps.testRandomUniformVariableDev
-          - TestBackendOps.testRandomNormalMean
-          - TestBackendOps.testVarSimple
-          - TestBackendOps.testWinograd
+        args: [backend_test.py]
```

Comment:
This is purely style, but why is this `args` line changed to `[item]` style when the other lines were changed to `- item` style?

Reply:
I have some yaml formatter that runs when I hit save. It decided that anything that doesn't fit within a single line should be split. In this case tho, it does fit and the formatter doesn't change it on me.